src/Pages/Platforms/platformTabs.tsx

```diff
@@ -31,34 +31,31 @@ export interface PlatformTabsProps extends RouteComponentProps {
  * Display tab bar and tab data for individual platforms
  * @param param0 React-Router props
  */
-export const PlatformTabs: React.SFC<PlatformTabsProps> = ({ match }) => {
+export const PlatformTabs: React.FunctionComponent<PlatformTabsProps> = ({ match }) => {
   const { id } = match.params
   const { path } = match
+  const unit_system = useUnitSystem()
 
   return (
-    <ErddapPlatformGetter platformId={id}>
+    <UsePlatform platformId={id}>
       {(platform_props) => (
         <React.Fragment>
           <Row style={{ paddingBottom: "1rem" }}>
             <Col>
               <Nav tabs={true}>
                 <ErddapObservedDropdown {...platform_props} />
-
                 <Tab to={paths.platforms.platform} path={path} name="Latest Conditions" id={id} />
-                <ForecastMetadataLoader>
-                  <React.Fragment>
-                    <ForecastDropdown platformId={platform_props.platform.id as string} />
-                  </React.Fragment>
-                </ForecastMetadataLoader>
-
+                <ForecastDropdown platformId={platform_props.platform.id as string} />
                 <ErddapMoreInfoDropdown {...platform_props} />
               </Nav>
             </Col>
           </Row>
 
           {/* Display our pages for the platform. */}
           <Switch>
-            <Route path={paths.platforms.all}>{(props) => <ErddapAllObservationsTable {...platform_props} />}</Route>
+            <Route path={paths.platforms.all}>
+              {(props) => <ErddapAllObservationsTable {...platform_props} unit_system={unit_system} />}
```

Comment:
Codacy found an issue: [Identifier 'platform_props' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)